docs/runtimes/serving-function.md

```diff
@@ -0,0 +1,21 @@
+(serving-function)=
+# Function of type `serving`
+
+Deploying models in MLRun uses the function type `serving`. You can create a serving function using the `set_function()` call from a notebook. 
+You can also import an existing serving function/template from the {ref}`function-hub`.
```

Comment:
This sentence is true for `job` functions as well (it really depends on the function you're importing). Would put it there as well.

Reply:
@theSaarco  - updated the job page